### ISSUE TITLE
Improve custom accuracy function support

### ISSUE THREAD

**Issue by paxcema** [mindsdb/mindsdb_evaluator#7]
The objective is to be able to add them in `~/lightwood_modules` for better user experience.

**Comment by paxcema** [comment]
Won't fix for now. Plenty of supported metrics via sklearn.
